src/data/resolvers/motions.ts

```diff
@@ -1232,6 +1232,10 @@ export const motionsResolvers = ({
         };
       }
 
+      if (actionValues.name === 'createDecision') {
```

Comment:
Not specific to your work but in general i think that it all should be redone with enums...

Reply:
I agree. I can raise a separate issue to fix.